src/lib.rs

```diff
@@ -81,3 +81,6 @@ pub mod notes {
     #[doc(include = "../docs/notes-KZG10.md")]
     pub mod kzg10_docs {}
 }
+
+/// Bls12_381 Scalar structure used by plonk.
+pub use dusk_bls12_381::Scalar;
```

Comment:
I thought we discussed about exporting the whole crate via PLONK, not just the `Scalar`.

Also, we should do that for `JubJub` as well, no?

Reply:
Yep, I'll do it for JubJub on a sepparated PR once a couple of PRs land in jubjub crate. 

I was just going to export the `Scalar` types from both crates instead of the whole crate. But maybe you have other ideas.

Reply:
As we discussed – as now I think it's clear – the idea to me was to re-export the whole crate so everything use PLONK can use automatically the whole jubjub / bls's crates that PLONK uses, reducing the confusion / errors about versioning, and be sure that if you use the one PLONK exports, you're good to go.
So not only the Scalars.

Reply:
That will be done correctly in #256

Reply:
It has been removed. Will be done in #256